libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -65,10 +65,20 @@ public MicrosoftAppCredentials(string appId, string password)
         /// <param name="appId">The Microsoft app ID.</param>
         /// <param name="password">The Microsoft app password.</param>
         /// <param name="customHttpClient">Optional <see cref="HttpClient"/> to be used when acquiring tokens.</param>
-        public MicrosoftAppCredentials(string appId, string password, HttpClient customHttpClient)
+        public MicrosoftAppCredentials(string appId, string password, HttpClient customHttpClient) : this(appId, password, null, null) { }
```

Comment:
BUG. This is incorrect. If I read the code right, this should be 
: this (appId, password, null, customHttpClient) {}

Reply:
doh!